platforms/core-configuration/flow-services/src/main/kotlin/org/gradle/internal/flow/services/FlowParametersInstantiator.kt

```diff
@@ -58,7 +58,7 @@ class FlowParametersInstantiator(
         val problems = ImmutableList.builder<Problem>()
         inspection.propertyWalker.visitProperties(
             parameters,
-            object : ProblemRecordingTypeValidationContext(type, { Optional.empty() }, problemsService) {
+            object : ProblemRecordingTypeValidationContext(type, { Optional.empty() }) {
```

Comment:
no because it is currently only used to get the [additional data factory](https://github.com/gradle/gradle/blob/master/platforms/core-configuration/model-core/src/main/java/org/gradle/internal/reflect/ProblemRecordingTypeValidationContext.java#L71) stuff